App/VSTS/FollowButton.tsx

```diff
@@ -0,0 +1,66 @@
+/// <reference path="../../office.d.ts" />
+import * as React from 'react';
+import { Provider } from 'react-redux';
+import { changeFollow } from '../actionsEZ';
+import { connect } from 'react-redux';
+import { FollowStateTypes } from '../statesEZ';
+
+let buttonText: string = '';
+let isDisabled: boolean = false;
+let iconType: string = 'ms-Icon ms-Icon--eye';
+
+interface IFollowButtonProps {
+  dispatch?: any;
+  followState?: FollowStateTypes;
+}
+
+function mapStateToProps(state: any): IFollowButtonProps {
+  return {
+    followState: state.quickActionState.followState,
+  };
+}
+
+@connect(mapStateToProps)
+export class FollowButton extends React.Component<IFollowButtonProps, {}> {
+  public render(): React.ReactElement<Provider> {
+    console.log('got to followbutton, followstate: ' + this.props.followState);
+    switch (this.props.followState) {
```

Comment:
I don't like this switch case here just to convert the enum to string. I would rather change the enum values to what we want in the test as well and then just use the enum value as string instead of doing this switch case. Ping me if you don't understand how to do this